cmd/aro/update_role_sets.go

```diff
@@ -0,0 +1,174 @@
+package main
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/arm/policy"
+	"github.com/sirupsen/logrus"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/database"
+	"github.com/Azure/ARO-RP/pkg/env"
+	"github.com/Azure/ARO-RP/pkg/metrics/statsd"
+	"github.com/Azure/ARO-RP/pkg/util/encryption"
+	"github.com/Azure/ARO-RP/pkg/util/keyvault"
+)
+
+// 1 - Get env data from agent VMs (with getEnvironmentData)
```

Comment:
Why the 1,2 & 3 here and below? If there's some ordering as to how these should be called is that seems better placed in the place where they are called or in a general overview of the flow IMO

Reply:
It's more of a logical order of steps that I used to help guide writing / reading the script. I can remove code comments if you don't feel like they are needed, or I can remove the numbering to avoid ordering confusion.

Reply:
IMO the comments can go. The code is pretty clear for me

Reply:
done!